### PR TITLE
Update log-files.md

### PR DESCRIPTION
**Pull request by kellybos**
Few updates to make this article more helpful and less confusing.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/log-files.md [hunk 1]
```diff
@@ -43,18 +43,24 @@ Right-click on the Teams icon in the system tray and choose **Collect support fi
...
+  
 Media logging is turned off by default. To enable Media logging, users must turn on the option in the Teams client. Go to **Settings** > **General**, and select **Enable logging for meeting diagnostics (requires restarting Teams)**. The Teams client must be restarted for logging to begin.
 
+If a problem occurs with a specific meeting or live event, it's helpful to have the URL associated with the meeting. This provides additional information to help pinpoint the exact meeting or live event in the logs. This information can be collected from any participant for a meeting or from presenter or producer for a live event. This URL can be captured by hovering over the join URL and choosing **Copy Hyperlink**.
+
 > [!NOTE]
 > If Media logging is enabled, there will be additional files included in the Meeting folder which are necessary for investigating audio and video issues. If Media logging is not enabled, there will be a limited number of logs available.
+  
+> [!NOTE]
+> The debug logs were previouly collecting using the keyboard shortcuts below.These still function and will complete the same log capture as the **Collect support files** option.
```
Comment (JohanFreelancer9) [inline Teams/log-files.md:56]:
```suggestion
> The debug logs were previously collecting using the keyboard shortcuts below. These still function and will complete the same log capture as the **Collect support files** option.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @kellybos, please commit my suggested changes.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@CarolynRowe PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by CarolynRowe** [comment]
This branch has conflicts that I am unable to resolve.  Please contact the submitter.

**Pull request merged**